src/Views/Form/Templates/Classic/resources/views/receipt.php

```diff
@@ -1,110 +1,125 @@
 <?php
 
+use Give\Helpers\Form\Template;
 use Give\Receipt\DonationReceipt;
 use Give\Receipt\LineItem;
 use Give\Receipt\Section;
 use Give\Session\SessionDonation\DonationAccessor;
-use Give\Views\Form\Templates\Sequoia\Sequoia;
 use Give\Views\IframeContentView;
-use Give\Helpers\Form\Template as FormTemplateUtils;
-use Give_Payment as Payment;
 
-$donationSessionAccessor = new DonationAccessor();
-$donation                = new Payment( $donationSessionAccessor->getDonationId() );
-$options                 = FormTemplateUtils::getOptions();
+$template = Give()->templates->getTemplate();
+$receipt  = $template->getReceiptDetails( ( new DonationAccessor() )->getDonationId() );
+$option   = function ( $name ) use ( $template ) {
+	static $options = [];
 
-/* @var Sequoia $sequoiaTemplate */
-$sequoiaTemplate = Give()->templates->getTemplate();
-$receipt         = $sequoiaTemplate->getReceiptDetails( $donation->ID );
+	if ( empty( $options ) ) {
+		$options = Template::getOptions()[ 'donation_receipt' ];
+	}
 
-/* @var LineItem|null $pdfReceiptDownloadLinkDetailItem */
-$pdfReceiptLinkDetailItem = null;
+	if ( isset( $options[ $name ] ) ) {
+		return $options[ $name ];
+	}
+
+	return '';
+};
+
+$donorDashboardUrl = get_permalink( give_get_option( 'donor_dashboard_page' ) );
 
 ob_start();
 ?>
-<div class="give-receipt-wrap give-embed-receipt">
-	<div class="give-section receipt">
-		<?php if ( ! empty( $options['thank-you']['image'] ) ) : ?>
-			<div class="image">
-				<img src="<?php echo $options['thank-you']['image']; ?>"/>
+	<div class="give-receipt-classic">
```

Comment:
We can make this an `article` element since it could be standalone content. I’ll add the suggestion with the closing tag down below.

```suggestion
	<article class="give-receipt-classic">
```